# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/html/syntax/xmldecl/xmldecl-2.html.ini
# Commit: 3dfd3c94a105
# Full Hash: 3dfd3c94a105e095aada0b356f1106370de222d3
# Author: Henri Sivonen <hsivonen@hsivonen.fi>
# Date: 2021-12-07 15:51:17
# Regressor Bug: 1701828
# File Overlap Count: 10
# Description:
#   Bug 1701828 - meta charset rewrite. r=smaug
#   
#   Implements https://github.com/whatwg/html/issues/6962 . Improves performance
#   when <meta charset> occurs in head but after the first kilobyte and aligns
#   behavior better with WebKit and Blink.
# ==============================================================================

diff --git a/testing/web-platform/meta/html/syntax/xmldecl/xmldecl-2.html.ini b/testing/web-platform/meta/html/syntax/xmldecl/xmldecl-2.html.ini
deleted file mode 100644
--- a/testing/web-platform/meta/html/syntax/xmldecl/xmldecl-2.html.ini
+++ /dev/null
@@ -1,14 +0,0 @@
-bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1701828
-[xmldecl-2.html]
-  [Check encoding windows-1251, kilobyte-plus-one-after.htm]
-    expected: FAIL
-
-  [Check encoding windows-1251, kilobyte-plus-one-before.htm]
-    expected: FAIL
-
-  [Check encoding windows-1251, kilobyte-plus-one-after-trail.htm]
-    expected: FAIL
-
-  [Check encoding windows-1251, kilobyte-plus-one-before-trail.htm]
-    expected: FAIL
-